app/controllers/api/v1/tokens_controller.rb

```diff
@@ -139,6 +154,11 @@ def revoke
         resource: token,
       )
 
+      # expire session if we're revoking the current token
+      if token == current_token
+        cookies.delete(:session_id, domain: Keygen::DOMAIN, same_site: :none)
```

Comment:
Is all this really needed? Invalid sessions will be auto-expired after unsuccessful the authn.